src/child/index.js

```diff
@@ -22,16 +25,26 @@ require('script!../../node_modules/d3fc/dist/d3fc.bundle.min.js');
 require('script!../../node_modules/BitFlux/dist/bitflux.js');
 /* eslint-enable import/no-unresolved */
 
+const { store, parent } = window.opener;
+
 currentWindowService.ready(() => {
     const currentWindow = currentWindowService.getCurrentWindow();
-    const store = currentWindow.contentWindow.opener.store;
     const rootElement = document.getElementById('app');
 
     const windowStateService = new WindowStateService(currentWindow, store, rootElement);
     windowStateService.start();
 
+    window.store = store;
```

Comment:
I'm really not keen on storing the `store` as a global on the child windows (I'm also not keen in the parent, but it seemed a reasonable compromise to give the child access to the store). Is this even used?